cmd2/cmd2.py

```diff
@@ -249,7 +264,7 @@ def __init__(self, completekey: str = 'tab', stdin=None, stdout=None, *,
         self.last_result = None
 
         # Used by run_script command to store current script dir as a LIFO queue to support _relative_run_script command
-        self._script_dir = []
+        self._script_dir = []  # type: List[AnyStr]
```

Comment:
Should it be `str` or `AnyStr`?  Would there ever be a case where we assign a `bytes`?

Reply:
I don't know. Was just trying to give a type to an untyped variable. PyCharm auto-detected this field as being likely List[AnyStr] so I went with it.